ccmlib/cluster.py

```diff
@@ -209,7 +209,7 @@ def balanced_tokens_across_dcs(self, dcs):
 
     def remove(self, node=None):
         if node is not None:
-            if not node.name in self.nodes:
+            if node.name not in self.nodes:
```

Comment:
flake8 for ccm is... not a pretty picture. I figured "leave the place cleaner than when I got here".